### PR TITLE
fixed env var error: added cross-env

### PR DESCRIPTION
**Pull request by tednaaa**

### CODE REVIEW

File: example/webpack5/webpack.config.ts [hunk 1]
```diff
@@ -1,29 +1,29 @@
-import { resolve } from "path";
+import { resolve } from 'path';
```
Comment (retyui) [inline example/webpack5/webpack.config.ts]: Could you please discard these changes?

```
git checkout origin/master -- example/webpack5/webpack.config.ts
```

File: example/webpack5/dist/index.js [hunk 2]
```diff
@@ -99,7 +99,7 @@ __webpack_require__.r(__webpack_exports__);
...
+___CSS_LOADER_EXPORT___.push([module.id, ".foo {\r\n  width: 240px;\r\n}\r\n\r\n.bar {\r\n  width: 160px;\r\n}\r\n\r\n.header-main {\r\n  background: red;\r\n}\r\n\r\n.footer-main {\r\n  background: blue;\r\n}\r\n\r\n@media screen and (min-width: 768px) {\r\n  .foo {\r\n    width: 576px;\r\n  }\r\n  .bar {\r\n    width: 384px;\r\n  }\r\n}\r\n\r\n@media all and (-webkit-min-device-pixel-ratio: 1.5),\r\n  (min--moz-device-pixel-ratio: 1.5),\r\n  (-o-min-device-pixel-ratio: 1.5/1),\r\n  (min-device-pixel-ratio: 1.5),\r\n  (min-resolution: 138dpi),\r\n  (min-resolution: 1.5dppx) {\r\n  .header-main {\r\n    background: gold;\r\n    -webkit-background-size: auto auto;\r\n    -moz-background-size: auto auto;\r\n    background-size: auto auto;\r\n  }\r\n  .footer-main {\r\n    background: pink;\r\n    -webkit-background-size: auto auto;\r\n    -moz-background-size: auto auto;\r\n    background-size: auto auto;\r\n  }\r\n}\r\n", ""]);
```
Comment (retyui) [inline example/webpack5/dist/index.js]: Seem on you operation system (windows) used `r\n` instead `\n` to brake lines 

Could you please discard these changes?

```
git checkout origin/master -- example/webpack5/dist/index.js
```

Comment (tednaaa) [reply example/webpack5/dist/index.js]: nothing changes when I write this command, what am I doing wrong?

image

Comment (retyui) [reply example/webpack5/dist/index.js]:
```bash
# ssh
git remote add base git@github.com:retyui/group-css-media-queries-loader.git
# or https
git remote add base https://github.com/retyui/group-css-media-queries-loader.git

# then try
git checkout base/master -- example/webpack5/dist/index.js example/webpack5/webpack.config.ts
```

File: .github/workflows/nodejs.yml [hunk 3]
```diff
@@ -3,17 +3,20 @@ name: Code Quality
 on:
   push:
     paths:
-      - ".github/**"
-      - "yarn.lock"
-      - "**.js"
+      - '.github/**'
+      - 'yarn.lock'
+      - '**.js'
+    branches: [master]
```
Comment (retyui) [inline .github/workflows/nodejs.yml]: I think it better to remove filter `branches: [master]`



```yml
on:
  push:
    paths:
      - ".github/**"
      - "yarn.lock"
      - "**.js"
  pull_request:
    paths:
      - ".github/**"
      - "yarn.lock"
      - "**.js"
```

**Comment by retyui** [comment]
@and-bulk Everything looks good here. Thanks for fixing cross-platform compatibility

**Pull request merged**